sunpy/data/data_manager/storage.py

```diff
@@ -122,16 +122,12 @@ def __init__(self, path):
             self._setup()
 
     def _setup(self):
-        schema = ' text, '.join(self.COLUMN_NAMES) + ' text'
         with self.connection(commit=True) as conn:
-            # Do this in a try...except to prevent race conditions in the tests
```

Comment:
I wonder if we need to worry about this anymore in our test suite.
I assume that "IF NOT EXISTS" might not avoid this problem?

Reply:
I'm sorry, did you mean "might avoid this problem"?

IF NOT EXISTS should take care of the exception that's being handled, since it pertains to creating a table that already existed.

Reply:
Yeah, sorry, I did mean "might avoid this problem".